UefiCpuPkg/Universal/Acpi/S3Resume2Pei/S3Resume.c

```diff
@@ -1082,6 +1082,17 @@ S3RestoreConfig2 (
       Status = SmmAccess->Open ((EFI_PEI_SERVICES **)GetPeiServicesTablePointer (), SmmAccess, Index);
     }
 
+    //
+    // Get MP Services2 Ppi to pass it to Smm S3.
+    //
+    Status = PeiServicesLocatePpi (
+               &gEdkiiPeiMpServices2PpiGuid,
+               0,
+               NULL,
+               (VOID **)&MpService2Ppi
+               );
+    ASSERT_EFI_ERROR (Status);
+

```

Comment:
This patch is a bit confusing.
You can update the next patch to LocatePpi and use it and create a new patch to remove the extra PPI-locate.

Reply:
sounds good.